### ISSUE TITLE
Meta OpenAPI tags are being ignored being added to methods

### ISSUE THREAD

**Issue by sherodtaylor** [goadesign/goa#3148]
The `Meta` definition for overriding openapi tags are being ignored

from the docs
```
"openapi:tag:xxx" sets the OpenAPI object field tag xxx. Applicable to services and methods.

var _ = Service("MyService", func() {
    Method("MyMethod", func() {
        Meta("openapi:tag:Backend")
        Meta("openapi:tag:Backend:desc", "Description of Backend")
        Meta("openapi:tag:Backend:url", "http://example.com")
        Meta("openapi:tag:Backend:url:desc", "See more docs here")
        Meta("openapi:tag:Backend:extension:x-data", `{"foo":"bar"}`)
    })
})
```

design.go
```
package design

import (
	. "goa.design/goa/v3/dsl"
)

var _ = API("calc", func() {
	Title("cal")
	Description("cal")
	Server("calc", func() {
		Host("localhost", func() {
			URI("http://localhost:8000")
		})
	})
})

var _ = Service("calc stuff", func() {
	Description("calc stuff")

	Method("calc", func() {
		Meta("openapi:tag:Multiply")
		Meta("openapi:tag:Multiply:desc", "Should multiply the stuff")
		Meta("openapi:tag:Multiply:url", "https://docs.google.com")
		Meta("openapi:tag:Multiply:url:desc", "See more docs")

		Payload(func() {
			OneOf("operation", func() {
				Attribute("add", func() {
					Attribute("a", Int)
				})

				Attribute("subtract", func() {
					Attribute("b", Int)
				})

			})
			Example("add", func() {
				Description("operation to add")
				Value(Val{
					"add": Val{
						"a": 1,
						"b": 1,
					},
				})
			})
			Example("subtract", func() {
				Description("operation to subtract")
				Value(Val{
					"subtract": Val{
						"a": 1,
						"b": 1,
					},
				})
			})
		})

		Result(func() {
			Attribute("result", Int)
		})

		HTTP(func() {
			POST("/compute/quota")

			Response(StatusAccepted, func() {
				Body(func() {
					Attribute("result", Int)
				})
			})
		})
	})

	Files("/openapi.json", "./gen/http/openapi.json")
})

```

openapi3.yaml

```
openapi: 3.0.3
info:
    title: cal
    description: cal
    version: "1.0"
servers:
    - url: http://localhost:8000
paths:
    /compute/quota:
        post:
            tags:
                - calc stuff
            summary: calc calc stuff
            operationId: calc stuff#calc
            requestBody:
                required: true
                content:
                    application/json:
                        schema:
                            $ref: '#/components/schemas/CalcRequestBody'
                        examples:
                            add:
                                summary: add
                                description: operation to add
                                value:
                                    add:
                                        a: 1
                                        b: 1
                            subtract:
                                summary: subtract
                                description: operation to subtract
                                value:
                                    subtract:
                                        a: 1
                                        b: 1
            responses:
                "202":
                    description: Accepted response.
                    content:
                        application/json:
                            schema:
                                $ref: '#/components/schemas/CalcStuffcalcResponseBody'
                            example:
                                result: 6825552331577586910
    /openapi.json:
        get:
            tags:
                - calc stuff
            summary: Download ./gen/http/openapi.json
            operationId: calc stuff#/openapi.json
            responses:
                "200":
                    description: File downloaded
components:
    schemas:
        CalcRequestBody:
            type: object
            properties:
                operation:
                    type: object
                    properties:
                        Type:
                            type: string
                            description: |-
                                Union type name, one of:
                                - "add"
                                - "subtract"
                            example: add
                            enum:
                                - add
                                - subtract
                        Value:
                            type: string
                            description: JSON formatted union value
                            example: '"JSON"'
                    example:
                        Type: subtract
                        Value: '"JSON"'
                    required:
                        - Type
                        - Value
            example:
                subtract:
                    a: 1
                    b: 1
        CalcStuffcalcResponseBody:
            type: object
            properties:
                result:
                    type: integer
                    example: 3793862871819669726
                    format: int64
            example:
                result: 8399553735696626949
tags:
    - name: calc stuff
      description: calc stuff

```

**Comment by sherodtaylor** [comment]
@ernesto-jimenez @raphael any ideas :) things like `openapi:summary` override but i can't get the tags to override

**Comment by sherodtaylor** [comment]
any ideas?

**Comment by raphael** [comment]
Thank you for the nice repro, this is fixed in #3190. A few things to note:
* The `Meta("openapi:tag:xxx")` DSL must appear in HTTP expressions (under `API`, `Service` or `Method`).
* If used in `API` or `Service` the DSL defines tags (their description, URL etc.)
* If used in `Method` the DSL attached the tag to the method.

The `Meta` header comment has been updated to reflect the above.

Example:
```go
   Service("test service", func() {
		HTTP(func() {
                        // Define the `SomeTag` and `AnotherTag` OpenAPI tags
			Meta("openapi:tag:SomeTag:desc", "Endpoint description")
			Meta("openapi:tag:SomeTag:url", "Endpoint URL")
			Meta("openapi:tag:AnotherTag:desc", "Endpoint description")
			Meta("openapi:tag:AnotherTag:url", "Endpoint URL")
		})
		Method("test_endpoint", func() {
			HTTP(func() {
				Meta("openapi:tag:SomeTag") // Attach `SomeTag` to `test_endpoint`
				POST("/")
			})
		})
		Method("another_test_endpoint", func() {
			HTTP(func() {
				Meta("openapi:tag:AnotherTag") // Attach `AnotherTag` to `another_test_endpoint`
				POST("/")
			})
		})
	})
```
